### PR TITLE
Random

### PR DESCRIPTION
**Pull request by YCGreen**

### CODE REVIEW

File: src/main/java/green/fallingsand/Main.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) {
+        Sand sand = new Sand(50, 10);
+        sand.randomSand(50);
+        System.out.println(sand.toString());
+        try {
+            while (System.in.read() == 10) {
```
Comment (andrewoid) [inline src/main/java/green/fallingsand/Main.java]: What is 10?

File: src/main/java/green/fallingsand/Main.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                onClick(sand);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+
+    }
+
+    public static void onClick(Sand sand) {
```
Comment (andrewoid) [inline src/main/java/green/fallingsand/Main.java]: It's not really an onClick

**Review by andrewoid — COMMENTED** [review]
Have you found a crash?

**Comment by andrewoid** [comment]
You added files in the .idea directory that should be excluded by gitignore

**Comment by YCGreen** [comment]
This seems to be happening with all my projects. I’m not doing anything manually. Do you know how I can fix it so I don’t have to remove them every time?

**Comment by YCGreen** [comment]
> Have you found a crash?

I have not. But I will keep looking.

File: src/main/java/green/fallingsand/Main.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+public class Main {
+    public static int ENTER = 10;
+
+    public static void main(String[] args) {
+        Sand sand = new Sand(50, 10);
+        sand.randomSand(50);
+        System.out.println(sand.toString());
+        try {
+            while (System.in.read() == ENTER) {
```
Comment (andrewoid) [inline src/main/java/green/fallingsand/Main.java]: https://docs.oracle.com/javase%2F7%2Fdocs%2Fapi%2F%2F/java/awt/event/KeyEvent.html#VK_ENTER is better I think

**Review by andrewoid — COMMENTED** [review]
Post a screenshot please as well

**Comment by YCGreen** [comment]
Screenshot 2024-03-11 at 2 05 38 PM

File: src/main/java/green/fallingsand/Main.java (hunk 3 above)
Comment (andrewoid) [reply src/main/java/green/fallingsand/Main.java]: Make this change then I will approve it.

**Review by andrewoid — APPROVED** [review]
Merge this and close your other PR.

**Pull request merged**